function/spring-computer-vision-function/src/main/java/org/springframework/cloud/fn/computer/vision/ComputerVisionFunctionConfiguration.java

```diff
@@ -0,0 +1,187 @@
+/*
+ * Copyright 2020-2024 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.fn.computer.vision;
+
+import java.awt.image.RenderedImage;
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.util.function.BiFunction;
+import java.util.function.Function;
+import java.util.function.Supplier;
+
+import javax.imageio.ImageIO;
+
+import ai.djl.inference.Predictor;
+import ai.djl.modality.Classifications;
+import ai.djl.modality.cv.Image;
+import ai.djl.modality.cv.ImageFactory;
+import ai.djl.modality.cv.output.CategoryMask;
+import ai.djl.modality.cv.output.DetectedObjects;
+import ai.djl.modality.cv.output.Joints;
+import ai.djl.spring.configuration.DjlAutoConfiguration;
+import ai.djl.spring.configuration.DjlConfigurationProperties;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import org.springframework.boot.autoconfigure.AutoConfiguration;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.integration.support.MessageBuilder;
+import org.springframework.messaging.Message;
+
+/**
+ * Auto-configuration for TensorFlow Object Detection Function.
+ *
+ * @author Christian Tzolov
+ */
+@AutoConfiguration(after = DjlAutoConfiguration.class)
+@EnableConfigurationProperties({ DjlConfigurationProperties.class, ComputerVisionFunctionProperties.class })
+public class ComputerVisionFunctionConfiguration {
+
+	private static final Logger log = LoggerFactory.getLogger(ComputerVisionFunctionConfiguration.class);
+
+	@Bean
+	public ImageFactory imageFactory() {
+		return ImageFactory.getInstance();
```

Comment:
Do we really need to have this as a bean?

Cannot we just have a `private` property in this class?